hadoop-hdds/docs/config.yaml

```diff
@@ -37,3 +37,21 @@ markup:
   goldmark:
     renderer:
       unsafe: true
+security:
+  enableInlineShortcodes: false
+  exec:
+    allow:
+      - ^dart-sass-embedded$
+      - ^go$
+      - ^npx$
+      - ^postcss$
+    osEnv:
+      - (?i)^(PATH|PATHEXT|APPDATA|TMP|TEMP|TERM)$
```

Comment:
We can omit these defaults.

```suggestion
```

(You can verify that they are still part of the config by making a typo in `OZONE_VERSION`, running `hugo` to reproduce the original issue, and checking the "current security configuration" included in the error message.)